lib/color_spinor_pack.cu

```diff
@@ -278,6 +300,18 @@ namespace quda {
       genericPackGhost<Float,ghostFloat,order,Ns,precision_spin_color_mapper<Float,ghostFloat,Ns,768>::nColor>(ghost, a, parity, nFace, dagger, destination);
     } else if (a.Ncolor() == 1024) {
       genericPackGhost<Float,ghostFloat,order,Ns,precision_spin_color_mapper<Float,ghostFloat,Ns,1024>::nColor>(ghost, a, parity, nFace, dagger, destination);
+#ifdef NSPIN1
+    } else if (a.Ncolor() == 1536) { // staggered KD 24 -> 64 nvec
+      genericPackGhost<Float,ghostFloat,order,Ns,precision_spin_color_mapper<Float,ghostFloat,Ns,1536>::nColor>(ghost, a, parity, nFace, dagger, destination);
```

Comment:
FYI, added some compile guards, removed some instantiations that are unnecessary for Wilson or staggered.